designdocs.md

```diff
@@ -79,6 +80,19 @@ Obviously, we need specs too. But they often come later in the lifecycle, and ar
 
 We could capture a designdoc in a github issue, and this may in fact be a good convention. But a typical issue or bug has different content than what goes into a designdoc and sometimes you don’t want to parse all the discussion, you just want the conclusion.
 
+## Design Doc Process
+* Create new docs in the `Filecoin Community` > `Design Docs` folder.
+  * Check that you're using the GApps identity
+  * Use `DRAFT`, `IN REVIEW`, `ACCEPTED` to note status.
+* When you're ready, enable sharing for public comment
+  * Click "Share" button
+  * Click "Who has access" to open the advanced settings panel
+  * Choose "Anyone with the link can *comment*" from dropdown
```

Comment:
nit: just now I ran into what seems to be a new google UI which required I toggled a slider to the right of the "Link Sharing" header before being able to hit the drop down menu.  Also now the drop down menu says "People at Filecoin with the link can comment".  The existing instructions were more than good enough, just noting the UI change.

Reply:
Thanks, I fixed these 2 details and also added “As of April 2019...” as a marker/insurance against future UI changes.